cognite_toolkit/_cdf_tk/utils/safe_dict.py

```diff
@@ -0,0 +1,96 @@
+import threading
+from collections import UserDict
+from collections.abc import Iterator
+from typing import Any, Generic
+
+from cognite_toolkit._cdf_tk.utils.useful_types import T_ID, T_Value
+
+
+class ThreadSafeDict(UserDict, Generic[T_ID, T_Value]):
```

Comment:
Should the python file be called thread_safe_dict.py, to reflect that the "safe" is related to thread-safety.

Reply:
done